src/Cards.svelte

```diff
@@ -31,31 +28,26 @@
     flex-direction: row;
     flex-wrap: wrap;
   }
-  .loading {
-    padding: 3rem 0;
-  }
 </style>
 
-{#await getOrgProfiles}
-  <div class="loading">
-    <Loading center />
-  </div>
-{:then orgProfiles}
   <div class="list">
-    {#each orgProfiles as profile}
-      {#if orgMembers[profile.address]?.length}
-        <Card {profile} {config} path={`/orgs/${profile.nameOrAddress}`} members={orgMembers[profile.address]} />
-      {:else}
-        <Card {profile} {config} path={`/orgs/${profile.nameOrAddress}`} />
-      {/if}
+    {#each orgs as org}
+      {#await Profile.get(org.name ?? org.address, ProfileType.Minimal, config)}
+        <Card profile={{ address: org.address }} {config} path={`/orgs/${org.address}`} />
+      {:then profile}
+        {#if orgMembers[profile.address]?.length}
+          <Card {profile} {config} path={`/orgs/${profile.nameOrAddress}`} members={orgMembers[profile.address]} />
+        {:else}
+          <Card {profile} {config} path={`/orgs/${profile.address}`} />
+        {/if}
+      {/await}
     {/each}
 
     {#each profiles as profile}
-      <Card {profile} {config} path={`/users/${profile.nameOrAddress}`} />
+      <Card {profile} {config} path={`/users/${profile.address}`} />
```

Comment:
Why?

Reply:
Yeah this is a leftover, we can safely use `nameOrAddress`